Source/Android/jni/AndroidCommon/IDCache.h

```diff
@@ -7,6 +7,8 @@
 
 namespace IDCache
 {
+JavaVM* GetJavaVM();
```

Comment:
This shouldn't be needed. You can get the VM from the env.

Reply:
I cannot find any reference on such a method. The only API I found is `JNI_GetDefaultJavaVMInitArgs` but that doesn't seem to be working on Android.

Reply:
The method is called `GetJavaVM`.